main.go

```diff
@@ -277,9 +281,22 @@ func main() {
 		setupLog.Error(clientErr, "failed to create kubernetes clientset")
 	}
 
-	reviewer := rbac.NewReviewer(clientset)
 	ctx := ctrl.SetupSignalHandler()
 
+	if createOpenShiftDashboard {
+		err = openshiftDashboards.CreateOpenShiftDashboard(clientset)
+		if err == nil {
+			ctrl.Log.Info("OpenShift dashboard for OpenTelemetryCollector monitoring created")
+			context.AfterFunc(ctx, func() {
+				openshiftDashboards.DeleteOpenShiftDashboard(clientset, ctrl.Log)
```

Comment:
So, in effect, the dashboard only exists while the operator is running? What happens if we have multiple operator replicas?

Reply:
> So, in effect, the dashboard only exists while the operator is running? 

Yes

> What happens if we have multiple operator replicas?

It will be only created the first time. It will be removed if any of the replicas is removed... but I don't know how to handle that..